includes/Story_Renderer/Embed.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Class Embed
+ *
+ * @package   Google\Web_Stories\Story_Renderer
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+namespace Google\Web_Stories\Story_Renderer;
+
+use Google\Web_Stories\Model\Story;
+
+/**
+ * Class Embed
+ *
+ * @package Google\Web_Stories\Story_Renderer
+ */
+class Embed {
+	/**
+	 * Current post.
+	 *
+	 * @var Story Post object.
+	 */
+	protected $story;
+
+	/**
+	 * Height of image
+	 *
+	 * @var Int Height of image.
+	 */
+	protected $height;
+
+	/**
+	 * Width of image
+	 *
+	 * @var Int Width of image.
+	 */
+	protected $width;
+
+	/**
+	 * Align class.
+	 *
+	 * @var string
+	 */
+	protected $align;
+
+	/**
+	 * Embed constructor.
+	 *
+	 * @param Story  $story   Story Object.
+	 * @param int    $width   Width of image.
+	 * @param int    $height  Height of image.
+	 * @param string $align   Align Image. Default: none.
+	 */
+	public function __construct( $story, $width, $height, $align = 'none' ) {
+		$this->story  = $story;
+		$this->width  = $width;
+		$this->height = $height;
+		$this->align  = $align;
+	}
+
+	/**
+	 * Renders the block output in default context.
+	 *
+	 * @return string Rendered block type output.
+	 */
+	public function render() {
+		$url          = $this->story->get_url();
+		$title        = $this->story->get_title();
+		$poster       = ! empty( $this->story->get_poster_portrait() ) ? esc_url( $this->story->get_poster_portrait() ) : '';
+		$align        = sprintf( 'align%s', $this->align );
+		$margin       = ( 'center' === $this->align ) ? 'auto' : '0';
+		$player_style = sprintf( 'width: %dpx;height: %dpx;margin: %s', absint( $this->width ), absint( $this->height ), esc_attr( $margin ) );
```

Comment:
Let's keep the spaces like we had them before

```suggestion
		$player_style = sprintf( 'width: %dpx; height: %dpx; margin: %s', absint( $this->width ), absint( $this->height ), esc_attr( $margin ) );
```

Reply:
This isn't by mistake, I got a validation error by adding these spaces in.

Reply:
What validation?

Reply:
In javascript / gutenberg

Reply:
Gutenberg block validation only compares the output of  the `save()` method, so this change here wouldn't affect block validation.

I assume the block validation error you saw is because you added `margin` in the `save` method, which I flagged earlier to be risky.

Reply:
No, I got the valdation error before I added. I was getting repeated properties, of height and width with and without the spaces.